release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 3. | **Feature** | [**General Availability**] Support for Multi-AZ VMSS | **Brief Description**: AZ Spanning Virtual Machine Scale Sets (VMSS) has been in preview for a year becasue we did not have the ability to orchestrate VMSS updates, one AZ at a time until now.<br>We can have the ability to drive VMSS updates, sequentially 1 zone at a time by setting VMSSZonalUpgradeMode = "Hierarchical". This value can be changed at anytime.<br>**Documentation Reference**: Read more about it at -<ul><li>[Deploy an Azure Service Fabric cluster across Availability Zones](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cross-availability-zones)</li></ul> |
+| 4. | **Feature** | Added support for IHost, IHostBuilder and Minimal Hosting Model | **Brief Description**: IHost integration in KestrelCommunicationListener and HttpSysCommunicationListener was shipped in SDK v5.2.1363 in Preview. In Service Fabric 9.0, this is made generally available.<br>Add IHost and Minimal Hosting based web service samples in the documentation.<br>In addition to IWebHost/IWebHostBuilder, KestrelCommunicationListener and HttpSysCommunicationListener now support building ASP.NET Core web services using IHost/IHostBuilder.<br>.Net 6 introduced the Minimal Hosting model which is a more simplified and streamlined way of creating ASP.NET Core web applications. Minimal hosting model can also be used for creating SF ASP.NET Core services.<br>This was available in preview starting with v5.2.1363 of Microsoft.ServiceFabric.AspNetCore.Kestrel and Microsoft.ServiceFabric.AspNetCore.HttpSys packages. Starting Service Fabric v.9.0, this has been made generally available.<br>**Documentation Reference**: Read more about it at -<ul><li>[ASP.NET Core in Azure Service Fabric Reliable Services](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-reliable-services-communication-aspnetcore)</li><li>[IHost and Minimal Hosting integration](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-reliable-services-communication-aspnetcore#ihost-and-minimal-hosting-integration)</li></ul> |
```

Comment:
Add IHost and Minimal Hosting based web service samples in the documentation -> Is this supposed to be replaced by a sample or documentation link?